sdk/keyvault/keyvault/README.md

```diff
@@ -1,7 +1,114 @@
-# Azure SDK KeyVault Library
+# Azure SDK KeyVault Library for C
 
 Azure KeyVault library (`az_keyvault`) provides abstractions, and helpers for communicating with Azure KeyVault in the C programming language. This library follows the Azure SDK Design Guidelines for Embedded C.
```

Comment:
It's actually a KeyVault Keys (az_keyvault_keys) library as we do not support other features of keyvault. However, I thought we were going to make this a sample and not a formal library that we support externally.

Reply:
The readme can be updated library->sample when we move the package